mmpose/models/keypoint_heads/temporal_regression_head.py

```diff
@@ -94,10 +94,24 @@ def get_loss(self, output, target, target_weight):
         """
         losses = dict()
         assert not isinstance(self.loss, nn.Sequential)
-        if target_weight is None:
-            target_weight = target.new_ones(target.shape)
-        assert target.dim() == 3 and target_weight.dim() == 3
-        losses['reg_loss'] = self.loss(output, target, target_weight)
+
+        # trajectory model
+        if self.num_joints == 1:
```

Comment:
I suggest we use an argument (e.g. is_trajectory) to explicitly determine whether this head is for trajectory prediction.